Framework/MDAlgorithms/test/IntegratePeaksMD2Test.h

```diff
@@ -12,6 +12,7 @@
 #include "MantidAPI/Run.h"
 #include "MantidDataObjects/LeanElasticPeaksWorkspace.h"
 #include "MantidDataObjects/MDEventFactory.h"
+#include "MantidDataObjects/NoShape.h"
```

Comment:
What's this extra header for? Not sure this is required

Reply:
Good spot, that was used for a test I tried to write before discovering MaskBTP wouldn't get called - I have removed it.